tests/test_blog.py

```
@@ -97,6 +97,37 @@ def mock_path_side_effect(path_str):
             BlogDigestBuilder()
             assert temp_blogs_dir.exists()
     
+    def test_upload_api_v3_to_r2_method_exists(self):
+        """Test that the upload_api_v3_to_r2 method exists and can be called."""
+        builder = BlogDigestBuilder()
+        
+        # Verify the method exists
+        assert hasattr(builder, 'upload_api_v3_to_r2')
```

Ruff (0.12.2)

Use of `assert` detected

(S101)